src/main/resources/line.st

```diff
@@ -8,15 +8,12 @@ for (equipment in network.<collectionName>) {
       <if(countries1)>FiltersUtils.isLocatedIn(<countries1>, equipment.terminal1)<endif>
       <if((equipmentId || equipmentName || countries1) && countries2)>&& <endif><\\>
       <if(countries2)>FiltersUtils.isLocatedIn(<countries2>, equipment.terminal2)<endif>
-      <if((equipmentId || equipmentName || countries1 || countries2) && nominalV1Type)>&& <endif><\\>
-      <if(nominalV1Equality)>FiltersUtils.isEqualityNominalVoltage(equipment.terminal1, <nominalV1>)<endif><\\>
-      <if(nominalV1Range)>FiltersUtils.isRangeNominalVoltage(equipment.terminal1, <minNominalV1>, <maxNominalV1>)<endif>
-      <if((equipmentId || equipmentName || countries1 || countries2 || nominalV1Type) && nominalV2Type)>&& <endif><\\>
-      <if(nominalV2Equality)>FiltersUtils.isEqualityNominalVoltage(equipment.terminal2, <nominalV2>)<endif><\\>
-      <if(nominalV2Range)>FiltersUtils.isRangeNominalVoltage(equipment.terminal2, <minNominalV2>, <maxNominalV2>)<endif>
-      <if((equipmentId || equipmentName || countries1 || countries2 || nominalV1Type || nominalV2Type) && substationName1)>&& <endif><\\>
+      <if((equipmentId || equipmentName || countries1 || countries2) && nominalVType)>&& <endif><\\>
+      <if(nominalVEquality)>FiltersUtils.isEqualityNominalVoltage(equipment.terminal1, <nominalV>)<endif><\\>
+      <if(nominalVRange)>FiltersUtils.isRangeNominalVoltage(equipment.terminal1.nominalVoltage, <minNominalV>, <maxNominalV>)<endif>
```

Comment:
isRangeNominalVoltage(equipment.terminal1, \<minNominalV\>, \<maxNominalV\>)  (to be homogeneous with isEqualityNominalVoltage just above)

Reply:
yes